libnavigation-core/src/main/java/com/mapbox/navigation/core/routeoptions/RouteOptionsUpdater.kt

```diff
@@ -144,6 +147,12 @@ class RouteOptionsUpdater {
                             }
                         }
                     )
+                    .apply {
+                        if (rerouteOptions != null && rerouteOptions.avoidManeuverTime != 0) {
+                            val meters = location.speed/rerouteOptions.avoidManeuverTime
+                            avoidManeuverRadius(meters.toInt())
```

Comment:
This has to be clamped to <0, 1000> to avoid response errors (let's add a test as well).